charts/ceph-csi-cephfs/templates/csidriver-crd.yaml

```diff
@@ -6,4 +6,6 @@ spec:
   attachRequired: false
   podInfoOnMount: false
   fsGroupPolicy: {{ default "File" .Values.CSIDriver.fsGroupPolicy }}
+{{- if semverCompare ">=1.25-0" .Capabilities.KubeVersion.GitVersion }}
```

Comment:
`.Capabilities.KubeVersion.GitVersion` seems to be depcreated in Helm v3?
Can we use `Capabilities.KubeVersion.Version`?

ref - https://helm.sh/docs/chart_template_guide/builtin_objects/

Reply:
Hi @iPraveenParihar I tested that if it works with `GitVersion`, and saw it in many helm charts, but yeah, I can change this, np.